scripts/spaceCart/spaceCartComponent.js

```diff
@@ -0,0 +1,36 @@
+
+/// get current facility mineral 
+
+// import transaction module 
+// 
+// a function with an if statement that invokes a function that renders the facility mineral if it is not undefined 
+
+import { getFacilityMineral, getPlanetMineral } from "../transaction.js"
+
+export const getSpaceCartHTML = async () => {
+    const facilityMineral = getFacilityMineral()
+    const currentPlanet = getPlanetMineral()
+
+    //get facilityMinerals
+    //get planets
+
+    //get current facilityMineralId from transaction
+    //get current planet from transaction
+
+    //find facilityMineral instance in facilityMinerals
+    //find planet instance in planets
+
+
+    let spaceCartHTML = `<div class="space-cart">
+     <h3 class="space-cart--header">Space Cart</h3>`
+    if (facilityMineral != null) {
+        spaceCartHTML += `<p> 1 ton of ${facilityMineral.mineral.name} from ${currentPlanet.name}</p>`
```

Comment:
currentPlanet is a planet mineral, not a planet and does not have a name attribute. I think you're looking for `currentPlanet.planet.name`. Side note, calling this variable `currentPlanet` is confusing since it's a planet mineral, not a planet, so you my want to change that name.